extensions/vertx-core/deployment/src/main/java/io/quarkus/vertx/core/deployment/VertxCoreProcessor.java

```diff
@@ -66,11 +71,18 @@ IOThreadDetectorBuildItem ioThreadDetector(VertxCoreRecorder recorder) {
     @Record(value = ExecutionTime.RUNTIME_INIT)
     CoreVertxBuildItem build(VertxCoreRecorder recorder,
             LaunchModeBuildItem launchMode, ShutdownContextBuildItem shutdown, VertxConfiguration config,
+            List<VertxOptionsConsumerBuildItem> vertxOptionsConsumers,
             BuildProducer<SyntheticBeanBuildItem> syntheticBeanBuildItemBuildProducer,
             BuildProducer<ServiceStartBuildItem> serviceStartBuildItem) {
 
+        Collections.sort(vertxOptionsConsumers);
+        List<Consumer<VertxOptions>> consumers = new ArrayList<>();
```

Comment:
Minor complaint: You can use a specific size here since it's just the size of `vertxOptionsConsumers`